nouveau_map_search.js

```diff
@@ -4842,3 +4844,15 @@ NouveauMapSearch.prototype.sFromLatLngToPixel = function (  in_Latng,
 
     return ret;
     };
+
+/*******************************************************************************************/
+/**
+ \brief Sort By Key
+ */
+/*******************************************************************************************/
+NouveauMapSearch.prototype.sortByKey = function (array, key) {
+    return array.sort(function(a, b) {
+        var x = a[key].toLowerCase(); var y = b[key].toLowerCase();
```

Comment:
It'd be a good idea to validate that a key actually exists before trying to call .toLowerCase on it
```
var x = a[key] === undefined ? '' : a[key].toLowerCase();
var y = b[key] === undefined ? '' : b[key].toLowerCase();
```

Reply:
cool got it. this shouldn't ever happen but def good practice.